action.js

```diff
@@ -51,14 +51,14 @@ const getBodyFromGitHubContext = (team_id, github) => {
   // Defines the named event in terms of a set of matching values embedded in
   // the github context object.
   const event_defs = [
-    { ctoai_event:"Change Initiated",
+    { event:"deployment succeeded",
       conditions: [
         [ ["context","eventName"], "pull_request" ],
         [ ["context","payload","pull_request","base","ref"], "master" ],
-        [ ["context","payload","action"], "opened" ]
+        [ ["context","payload","action"], "merged" ]
```

Comment:
merged is not an action, you can use `closed`, `opened`, `synchonized`, `edited`

Reply:
Yup, I know. Needs to be updated still.